mne_bids_pipeline/scripts/preprocessing/_04b_run_ssp.py

```diff
@@ -88,23 +89,23 @@ def run_ssp(*, cfg, subject, session, in_files):
         projs[kind] = []
         if not any(n_projs[kind]):
             continue
-        proj_epochs = epochs_fun[kind](raw, ch_name=ch_name[kind])
+        proj_epochs = epochs_fun[kind](
+            raw, ch_name=ch_name[kind], decim=cfg.decim)
```

Comment:
I consider it a bugfix that we now apply `decim=cfg.decim` here. We probably should have done it before

Reply:
I am not sure. Doc says: https://github.com/mne-tools/mne-bids-pipeline/blob/main/mne_bids_pipeline/config.py#L738
and here it's epochs. Now you could argue that decim should apply to raw not epochs but I don't think it's a bug here.

Reply:
Here I mean that we weren't EOG/ECG decimating epochs, even though we should have been (for consistency with task epochs). So I think this PR is more consistent.

Elsewhere this PR fixes a bug where we were decimating epochs *twice*, meaning you end up fitting on the uncleaned, unrejected epochs (good) with decim-squared decimation applied instead of just decim (bad).